pkg/wasm/main.go

```diff
@@ -24,6 +27,26 @@ func getSource() string {
 	return getString(ptr, length)
 }
 
+type yielder struct {
```

Comment:
perhaps call this `sleepingYielder` or something?

Reply:
done.